Packages/com.llealloo.audiolink/Runtime/Scripts/YtdlpPlayer.cs

```diff
@@ -156,7 +156,7 @@ public static void LocateYtdlp()
             
             // Check for yt-dlp in VRC application data first
             _ytdlpPath = string.Join("\\", splitPath.Take(splitPath.Length - 2)) + "\\VRChat\\VRChat\\Tools\\yt-dlp.exe";
-            #elif UNITY_EDITOR_LINUX
+            #else
```

Comment:
this path is now MAC or LINUX is that intentional?

Reply:
ah it looks like it is intentional

Reply:
yes. its a decent first guess for both